VRDR/ValueSets.cs

```diff
@@ -0,0 +1,181 @@
+namespace VRDR
+    {
+         /// <summary> ValueSet Helpers </summary>
+         public static class ValueSets
+    {
+            /// <summary> EducationLevel </summary>
+            public static class EducationLevel {
+                /// <summary> Codes </summary>
+                public static string[,] Codes = {
+                { "PHC1448", "8th grade or less", VRDR.CodeSystems.PH_PHINVS_CDC },
```

Comment:
Indenting is off from lines 10 through 38 (should have an additional 4 spaces), and it's the same for each section